src/frontend/ocamlmerlin/new/new_commands.ml

```diff
@@ -341,6 +341,40 @@ match expression that contains the cursor
     end
   ;
 
+  command "jump-match-next-case"
```

Comment:
I think it would be much easier to adapt editors plugins by adding a new "target" to the existing ` jump` command instead of adding to very specific new commands.

Reply:
Thank you for the review. I think the new commits have merged the logic with the pre-existing `jump` command